LICENSE

```diff
@@ -1,3 +1,12 @@
+Copyright (c) 2021-present PrairieLearn, Inc.
+
+Portions of this software are licensed as follows:
+
+* All content that resides under the "ee/" directory of this repository, if that directory exists, is licensed under the license defined in "ee/LICENSE".
+* All client-side JavaScript (when served directly or after being compiled, arranged, augmented, or combined), is licensed under the "MIT Expat" license.
```

Comment:
This same item appears in Gitlab's license - I don't know if it matters if this is "MIT Expat" or "AGPL"? For reference, their _entire_ codebase (minus EE) is also licensed under the MIT license.

Reply:
```suggestion
* All client-side JavaScript (when served directly or after being compiled, arranged, augmented, or combined), is licensed under the "GNU Affero General Public License".
```

Reply:
I think we should just drop this client-side clause entirely.

Reply:
Matt and I talked offline and we're going to keep this in.

Reply:
But with AGPL, right?

Reply:
Correct, I changed this to AGPL!